CHANGELOG.md

```diff
@@ -8,12 +8,17 @@ All notable changes to this extension will be documented in this file.
 
 - "Open Settings" command to quickly access the Confluent for VS Code extension settings.
 
+### Changed
+
+- Topic -> Schema correlation now recognizes schemas using `TopicRecordNameStrategy` in addition to
+  `TopicNameStrategy`, [issue #298](https://github.com/confluentinc/vscode/issues/298).
+
 ## 0.18.0
 
 ### Added
 
 - New context menu item "View Latest Schema Version(s)" to quickly open the highest versioned value
-  and / or key schemas for a CCloud topic, based on TopicNameStrategy,
```

Comment:
I'd leave this in since the TopicRecordNameStrategy support won't be released until 0.19.x